lib/helpers-get-urls/get-urls-from-file-online.js

```diff
@@ -0,0 +1,58 @@
+'use strict'
+
+const path = require('path')
+const https = require('https')
```

Comment:
This will only support https and not http. Even though most sites are, and should be, https it is not much work to support both protocols.

Reply:
Got it... Hmm, do you know if ```http``` package supports file fetching from both ```http```/```https``` protocols? (even better if it supports ```ftp``` too)
Because I just need a single solution that will take care of fetching file from any web server, regardless of the protocol.

Reply:
Nope, two different protocols.

I've solved it like this in other modules (for http/https) https://github.com/zrrrzzt/url-status-code/blob/master/lib/check-url-status.js#L21

Reply:
I see, I like it!